common/src/main/java/com/cinemamod/mcef/ModScheme.java

```diff
@@ -0,0 +1,116 @@
+package com.cinemamod.mcef;
+
+import java.io.IOException;
+import java.io.InputStream;
+
+import com.mojang.logging.LogUtils;
+import org.cef.callback.CefCallback;
+import org.cef.handler.CefResourceHandler;
+import org.cef.misc.IntRef;
+import org.cef.misc.StringRef;
+import org.cef.network.CefRequest;
+import org.cef.network.CefResponse;
+
+// https://github.com/CinemaMod/mcef/blob/master-1.19.2/src/main/java/net/montoyo/mcef/example/ModScheme.java
+public class ModScheme implements CefResourceHandler {
+    private String contentType = null;
+    private InputStream is = null;
+
+    String url;
+
+    public ModScheme(String url) {
+        this.url = url;
+    }
+
+    private static final org.slf4j.Logger LOGGER = LogUtils.getLogger();
```

Comment:
How come we use Mojang LogUtils here?

Reply:
is there something you want me to use instead?
The ModScheme is a port of [the old ModScheme.java](https://github.com/CinemaMod/mcef/blob/master-1.19.2/src/main/java/net/montoyo/mcef/example/ModScheme.java) from 1.19.2, which was probably a copy of the 1.12 one